decidim-core/lib/decidim/core.rb

```diff
@@ -7,4 +7,25 @@ module Decidim
   autoload :TranslatableAttributes, "decidim/translatable_attributes"
   autoload :FormBuilder, "decidim/form_builder"
   autoload :DeviseFailureApp, "decidim/devise_failure_app"
+  include ActiveSupport::Configurable
+
+  # Loads seeds from all engines.
+  def self.seed!
+    Rails.application.railties.select do |railtie|
+      railtie.respond_to?(:load_seed) && railtie.class.name.include?("Decidim::")
+    end.each(&:load_seed)
+  end
```

Comment:
I think so, yes. The meta-gem should only be considered a bundle with some facilities like application generation & such. Do you think they should live on `decidim` instead of `decidim-core`?